platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -559,6 +559,13 @@ class Godot(private val context: Context) : SensorEventListener {
 		}
 	}
 
+	/**
+	 * Configuration change callback
+	*/
+	fun onConfigurationChanged(newConfig: Configuration) {
+		GodotLib.themeChanged()
```

Comment:
`onConfigurationChanged` is fired for every config changes we override, and we override [quite a few](https://github.com/godotengine/godot/blob/0bcc0e92b3f0ac57d4c4650722f347593a258572/platform/android/java/app/AndroidManifest.xml#L42) so this approach is not sufficient as it'll create a lot of invalid `themeChanged` callbacks.

To report changes for the theme only, we'll need to store the night mode flags on startup (e.g: [in `Godot::onCreate(...)` after the activity is retrieved](https://github.com/godotengine/godot/blob/0bcc0e92b3f0ac57d4c4650722f347593a258572/platform/android/java/lib/src/org/godotengine/godot/Godot.kt#L190), and compare the stored value with the new value retrieved from the new `Configuration` instance provided by `onConfigurationChanged`.

[Here](https://stackoverflow.com/a/44170179) is an example from SO.

Reply:
Done.